src/Components/SmartComponents/ClusterDetail/ClusterDetailPage.js

```diff
@@ -25,15 +31,44 @@ const ClusterDetailPage = () => {
     <ErrorHandler error={error}>
       <ClusterDetailPageHeader />
       <Main>
-        <TextContent>
-          <Text component={TextVariants.h2} className="pf-u-mb-md">
-            CVEs
-          </Text>
-        </TextContent>
-        <ClusterDetailTable />
+        {activeTab == CLUSTER_DETAIL_TABS.cves ? (
+          <Tabs className="pf-m-light pf-v5-c-table" activeKey={0}>
```

Comment:
> if these 2 tab components are here for the same tab contents (CVEs, Exposed images), I believe that there is a way to use only one of them. Am I right?

I just created a new component `TableTabs` to isolate this component, make it reusable and avoid code duplication. I wish I could also isolate the separate tabs into separate component instead of writing `renderTabs`, but Patternfly does not allow `Tab` components which are not a direct descendants of `Tabs` component.

> This is also a pretty good candidate to be shared by making it a bit more abstract and pushing to the pf-component-groups package.

This is a good thing to think about, it would depend on how the new Data view table abstraction will work. This paradigm of 1 router per tab is not followed by all Insights apps -- we would need to discuss the pros and cons of using URL path (route) vs. URL parameter to distinguish active tabs.

Reply:
Yup, I think that the first step is to create a shared component with the right architecture and then we can bring other applications to consistency one by one.